EndlessClient/UIControls/CreateCharacterControl.cs

```diff
@@ -72,5 +72,22 @@ private static CharacterRenderProperties GetDefaultProperties()
         {
             return new CharacterRenderProperties.Builder { HairStyle = 1 }.ToImmutable();
         }
+
+        public void UpdateRenderProperties(int hairStyle, int hairColor, int race, int gender, int armorgraphic, int weapongraphic, int bootgraphic, int shieldgraphic)
```

Comment:
Instead of adding an `UpdateRenderProperties` method, add a constructor overload (and factory function overload) that takes in the current character's render properties instead of using default properties. Then you can just construct the object directly in `BarberDialog` from the main character's render properties.

It looks like barber dialog doesn't need to arbitrarily set the render properties, and I'd prefer not to have a mutating method like this on the control's interface.

For clarify, I'm thinking like this:
```csharp
public CreateCharacterControl(ICharacterRendererFactory characterRendererFactory)
    : this(GetDefaultProperties(), characterRendererFactory) { }

public CreateCharacterControl(CharacterRenderProperties renderProperties, ICharacterRendererFactory factory)
    : base(Character.Default.WithRenderProperties(renderProperties), factory
{
    SetSize(99, 123);
    _lastPosition = Vector2.Zero;
}
```